package.json

```diff
@@ -46,11 +47,9 @@
     "null-loader": "3.0.0",
     "postcss": "7.0.16",
     "postcss-custom-properties": "6.3.1",
-    "react": "15.6.2",
-    "react-addons-test-utils": "15.6.2",
-    "react-dimensions": "1.3.1",
+    "react": "17.0.1",
```

Comment:
Don't we need an update to the peer dependencies?

Reply:
Updated